CHANGELOG.md

```diff
@@ -13,6 +13,49 @@ _Note: Gaps between patch versions are faulty, broken or test releases._
 
 See [CHANGELOG - 6to5](CHANGELOG-6to5.md) for the pre-4.0.0 version changelog.
 
+## v6.11.0 (2016-06-27)
+
+- In this release, there's now an `es2016` preset which includes the exponentiation operator (thanks for ([@ysmood](https://github.com/ysmood)) for giving us the npm package).
+- The `trailing-function-commas` plugin has been moved from `stage-2` preset to `stage-3` preset.
+- `babel-plugin-transform-react-jsx-self` has been added to the `react` preset in development mode. There is a new dev warning being added and this plugin `__self={this}` JSX attribute to all JSX elements
```

Comment:
@jimfb @spicyj Should I do a different description for this (I didn't have much context from the PR other than some warning)